Bottom: 28ededde9c241b9f6358a4001ae4552e93909e18
Top:    6ff4c6643aa76df7b679da21b73af5b1bc87f848
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:06:58 +0000

Refresh of remove-debug-from-hairline-0

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5b8bb06..b726d7b 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2408,8 +2408,6 @@ add_dummy_descriptors_at_point (Vector point, char poly, CVCList * list, POLYARE
   POLYAREA *b = bfst;
   PLINE *cur;
 
-  pcb_fprintf (stderr, "LOOKING FOR VERTICES AT POINT (%$mn, %$mn)\n", point[0], point[1]);
-
   assert (b != NULL);
   do
     {
